pyaerocom/region.py

```diff
@@ -367,7 +367,7 @@ def find_closest_region_coord(lat, lon, regions=None):
     matches = get_regions_coord(lat, lon, regions)
 
     if len(matches) == 1:
-        return matches[0]
+        return matches[0], matches
```

Comment:
You're changing the type of the return from `str` to `tuple[str, list[str]]`.
I would create a new data structure for the result. Something based on `typing.NamedTuple`, like this:

``` python
from __future__ import annotations
[...]
from typing import NamedTuple
[...]

class ClosestRegion(NamedTuple):
    best: str
    matches: list[str]

def find_closest_region_coord(
    lat: float,
    lon: float,
    regions: dict[str, Region] | None = None
) -> ClosestRegion:
    [...]
    return ClosestRegion(best, matches)
```

This way you get attribute access to the `find_closest_region_coord` resuts, e.g.

``` python
reg = find_closest_region_coord(lon, lat, regions=regions)
# closest region ID
reg.best
# all regions applicable
reg.matches 
```